### PR TITLE
Fix error messages to compression policy

### PR DESCRIPTION
**Pull request by k-rus**
The policy is called compression now and this fixes error messages.

### CODE REVIEW

File: tsl/src/bgw_policy/compression_api.c [hunk 1]
```diff
@@ -149,7 +149,7 @@ policy_compression_add(PG_FUNCTION_ARGS)
 		ts_cache_release(hcache);
 		ereport(ERROR,
 				(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-				 errmsg("can add compress_chunks policy only on hypertables with compression "
+				 errmsg("can add compression policy only on hypertables with compression "
```
Comment (erimatnor) [inline tsl/src/bgw_policy/compression_api.c:152]: I find the formulation here can be improved. Realize it was there before, but maybe we can take the opportunity to improve this:


```
errmsg("compression not enabled on hypertable \"%s\"", ....),
errhint("Enable compression before adding a compression policy.")
```

File: tsl/src/bgw_policy/compression_api.c [hunk 2]
```diff
@@ -191,7 +191,7 @@ policy_compression_add(PG_FUNCTION_ARGS)
 		{
 			ts_cache_release(hcache);
 			elog(WARNING,
-				 "could not add compress_chunks policy due to existing policy on hypertable with "
+				 "could not add compression policy due to existing policy on hypertable with "
```
Comment (erimatnor) [inline tsl/src/bgw_policy/compression_api.c:194]: Same here:

```
errmsg("compression policy already exist on hypertable \"%s\"", ...),
errhint("Remove the existing policy before adding a new one.")
```

File: tsl/src/bgw_policy/compression_api.c (hunk 1 above)
Comment (k-rus) [reply tsl/src/bgw_policy/compression_api.c]: Improved

File: tsl/src/bgw_policy/compression_api.c (hunk 2 above)
Comment (k-rus) [reply tsl/src/bgw_policy/compression_api.c]: Improved

**Pull request merged**